.github/workflows/label-deploy-failed.yml

```
@@ -3,39 +1,44 @@
+      github.event.issue.pull_request &&
       contains(github.event.comment.body, 'Deploy Preview')
+
     steps:
-      - name: Add 'Deploy-Failed'
-        if: contains(github.event.comment.body, 'failed')
+      - name: Check out the repository
+        uses: actions/checkout@v3
+
+      - name: Check deploy status
+        id: check-status
         uses: actions/github-script@v6
```

actionlint (1.7.4)

the runner of "actions/github-script@v6" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)